src/multiselect/use-multiselect.tsx

```diff
@@ -209,21 +202,6 @@ export function useMultiselect({
     renderHighlightedAriaLive,
   });
 
-  const tokens: TokenGroupProps['items'] = selectedOptions.map(option => ({
```

Comment:
Unused code, there is a duplicate of this https://github.com/cloudscape-design/components/blob/f7988a7d2a9d43b9e03156b084b3220189b693f5/src/multiselect/internal.tsx#L115-L128